### DISCUSSION TITLE
Pre-Recorded Transciption using Hindi language - Sentiment is wrong

### DISCUSSION THREAD

**Discussion by devb** [deepgram/community#382] [General help] [answered]
### Which Deepgram product are you using?

Deepgram SDKs

### Details

We're playing with DG's Transcription using your node js package to explore DG's speech-to-text service. So, later on we can go for the paid one. However, the "sentiment" is not coming properly for the "Hindi" language call transcription. It is doing good in terms of transcribing the audio in mix English and Hindi language. But the sentiment that is coming "word" wise and that too is not accurate. It should actually coming for the paragraph/utterance based instead of word to word and should be matched based on the whole paragraph/utterance by the speaker. Speaker diarization's accuracy is also 50%-50% ratio.

Since we're getting error using "phonecall" model for Hindi language. We're using "general" model as per the error suggestion.

**Comment by lukeocodes** [comment, marked as answer]
Hi @devb, the `phonecall` model is not available on Hindi. We're glad you're find the English/Hindi in stream mix valuable.

**Reply by devb** [reply]
Yes, like I said.. I was using "phonecall" model but because of these problems.. I've shifted to "general" model.. as mentioned in my message above. But diarization is not at all good along with the sentiments for each paragraph phrases.
